src/lib/compress_hw.c

```diff
@@ -324,6 +324,7 @@ static int compress_hw_write(void *data, const void *buf, size_t size)
 		cbuf += written;
 		total += written;
 	}
+
```

Comment:
drop this superfluous change

Reply:
ok, thanks for review,
will remove this change.

Thanks,
mohs.